shell/browser/api/electron_api_menu_mac.mm

```diff
@@ -177,8 +177,11 @@
 }
 
 // static
-gin_helper::WrappableBase* Menu::New(gin::Arguments* args) {
-  return new MenuMac(args);
+gin::Handle<Menu> Menu::New(gin::Arguments* args) {
+  auto handle =
+      gin::CreateHandle(args->isolate(), static_cast<Menu*>(new MenuMac(args)));
+  gin_helper::CallMethod(args->isolate(), handle.get(), "_init");
```

Comment:
`_init` was a notion from `native_mate` https://github.com/electron/native-mate/blob/master/native_mate/wrappable.cc#L41 , if this is being maintained in `gin` world as well, then can this be made a part of `gin_helper::constructible` ?

Reply:
I'd prefer to avoid it completely, though in this case removing it would have meant a more major refactor than I wanted to embark on. For now I'd like to put it in on a case-by-case basis and see how we go. It's a 1-liner to call it so it's not too much trouble.